tools/tests/merger/fixtures/extractor/opensearch/expected_1.3.yaml

```diff
@@ -12,33 +12,33 @@ paths:
       x-version-added: '1.0'
       description: Returns information about nodes in the cluster.
       parameters:
-        - $ref: '#/components/parameters/nodes.info::path.id'
+        - $ref: '#/components/parameters/nodes.info__path.id'
       responses:
         '200':
-          $ref: '#/components/responses/nodes.info@200'
+          $ref: '#/components/responses/nodes.info.200'
   /index:
     get:
       operationId: get.0
       responses:
         '200':
-          $ref: '#/components/responses/info@200'
+          $ref: '#/components/responses/info.200'
         '201':
-          $ref: '#/components/responses/info@201'
+          $ref: '#/components/responses/info.201'
         '404':
-          $ref: '#/components/responses/info@404'
+          $ref: '#/components/responses/info.404'
         '500':
-          $ref: '#/components/responses/info@500'
+          $ref: '#/components/responses/info.500'
         '503':
-          $ref: '#/components/responses/info@503'
+          $ref: '#/components/responses/info.503'
         removed-2.0:
-          $ref: '#/components/responses/info@removed-2.0'
+          $ref: '#/components/responses/info.removed-2.0'
           x-version-removed: '2.0'
         removed-2.0-refs:
-          $ref: '#/components/responses/info@removed-2.0-refs'
+          $ref: '#/components/responses/info.removed-2.0-refs'
         added-1.3-removed-2.0:
-          $ref: '#/components/responses/info@added-1.3-removed-2.0'
```

Comment:
This gets tricky with operation-group in non-root namespaces, say `indices.create`. We will end up with `indices.create.removed-2.0`. Visually it's hard to tell which is which. Is it a response for `indices` that's named `create.remove-2.0` or, a response for `indices.create` that's named `removed-2.0`.

This is not an issue on the technical term because the CodeGen does not parse the response names (at least the ones I work don't. @Xtansia can chime in), but we loses the visual cue with this change. I'm just stating a cost of this change. It's not a blocker.

We can also replace `@` with `___` --> `indices.create___removed-2.0`

Reply:
I went with this map.

```
      .replaceAll('::', '___')
      .replaceAll('@', '__')
      .replaceAll(':', '_')
```